src/components/LegalLines/Contact/Contact.js

```diff
@@ -32,21 +32,22 @@ const comps = {
   fr: (
     <div className="tm-imprint">
       <p>
-        Avete domande, proposte di miglioramento, segnalazioni di inesattezze o
-        siete interessati.
+        Vous avez des questions, des suggestions, des remarques sur des
+        inexactitudes ? Les cartes et plans vous intéressent? N’hésitez pas à
+        nos contacter.
       </p>
       <p>
-        <b>FFS SA</b>
+        <b>CFF SA</b>
         <br />
-        Infrastruttura – Impianti e tecnologia – Operation Center Tecnica –
-        Technical Management
+        Infrastructure – Installations et technologie – Operation Center
+        Technique – Technical Management
         <br />
         Fachbus Trafimage
         <br />
         Poststrasse 6<br />
         3072 Ostermundingen
         <br />
-        Svizzera
+        Suisse
```

Comment:
Some lines below, the domain in
`<a href="mailto:trafimage@ffs.ch">trafimage@ffs.ch</a>`
must must be `cff.ch`

You could solve it similar to [this](https://github.com/geops/trafimage-maps/blob/master/src/layerInfos/ShowcasesTopicInfo/ShowcasesTopicInfo.js#L74).